diagnostics/diagnostics_metrics.go

```diff
@@ -2,94 +2,98 @@ package diagnostics
 
 import (
 	"runtime"
+	"time"
 
 	"github.com/stripe/veneur/v14"
 )
 
-var (
-	gitSha                  string = "git_sha: " + veneur.VERSION
-	prevTotalMalloc         uint64 = 0
-	prevTotalFreed          uint64 = 0
-	prevTotalHeapAllocBytes uint64 = 0
-)
+var gitSha string = "git_sha: " + veneur.VERSION
```

Comment:
dogstatsd tag should not have space here

Reply:
done